tests/run/coverage_cmd_src_pkg_layout.srctree

```diff
@@ -0,0 +1,177 @@
+# mode: run
+# tag: coverage,trace
+
+"""
+PYTHON -m pip install .
```

Comment:
This line seems to be required (the test fails without it), but it modifies the current venv. A test should never do that. See the report in https://github.com/cython/cython/issues/5788